_episodes/05-r-telemetry-report-array.md

```diff
@@ -9,7 +9,7 @@ questions:
 
 ### Mapping GLATOS stations - Static map
 
-This section will use the Receivers CSV for the entire GLATOS Network.
+This section will use the Receivers CSV for the entire ACT Network.
```

Comment:
It's going to have a subset of ACT stations that are cleared to go public. We should also change the header from Mapping GLATOS stations - static map

Reply:
changes made